src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/impl/CometTranslationServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.ons.census.fwmt.outcomeservice.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.common.error.GatewayException;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.comet.HouseholdOutcome;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.CollectionCase;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.Event;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.InvalidAddress;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.OutcomeEvent;
+import uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.Payload;
+import uk.gov.ons.census.fwmt.outcomeservice.service.CometTranslationService;
+import uk.gov.ons.census.fwmt.outcomeservice.service.OutcomeService;
+
+import static uk.gov.ons.census.fwmt.outcomeservice.data.dto.rm.InvalidAddress.ReasonEnum.DERELICT;
+
+@Service
+public class CometTranslationServiceImpl implements CometTranslationService {
+
+  @Autowired
+  private OutcomeService outcomeService;
+
+  public void transformCometPayload(HouseholdOutcome householdOutcome) throws GatewayException {
+
+    OutcomeEvent outcomeEvent = new OutcomeEvent();
+    Payload payload = new Payload();
+    InvalidAddress invalidAddress = new InvalidAddress();
+    CollectionCase collectionCase = new CollectionCase();
+
+    collectionCase.setId(householdOutcome.getCaseId());
+    invalidAddress.setCollectionCase(collectionCase);
+    payload.setInvalidAddress(invalidAddress);
+    outcomeEvent.setPayload(payload);
+
+    buildOutcome(householdOutcome, outcomeEvent);
+
+    Event event = new Event();
+    event.setTransactionId("c45de4dc-3c3b-11e9-b210-d663bd873d93");
+
+    outcomeEvent.setEvent(event);
+    outcomeService.sendOutcome(outcomeEvent);
+  }
+
+  private void buildOutcome(HouseholdOutcome householdOutcome,
```

Comment:
Leave as is for now but...
In the back of my mind I have the idea how we had different convertors in Jobservice. It was a lot neater, but this has the extra complication of secondary reasons